warehouse/integrations/postgres/load.go

```diff
@@ -532,35 +532,32 @@ func (pg *Postgres) loadUsersTable(
 
 	// Deduplication
 	// Delete from users table if the id is present in the staging table
-	primaryKey := "id"
-	query = fmt.Sprintf(`
-		DELETE FROM
-		  %[1]q.%[2]q using %[3]q _source
-		WHERE
-		  (
-			_source.%[4]s = %[1]s.%[2]s.%[4]s
-		  );
-`,
-		pg.Namespace,
-		warehouseutils.UsersTable,
-		usersStagingTableName,
-		primaryKey,
-	)
+	if pg.enableDedup() {
```

Comment:
We weren't checking any flags for dedup here, just pinging you guys to make sure it makes sense to do so before I add the tests. @lvrach @achettyiitr

Reply:
Remove these checks for users tables.